### PR TITLE
Release 2.0.0

### PR DESCRIPTION
**Pull request by marrony**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,57 +5,35 @@
 
 ## How to Build
 
-### Visual Studio
+### Necessary Tools
```
Comment (erickpintor) [inline README.md]: I'd say "Requirements" instead of "Necessary Tools". It is what we use in other places.

File: README.md [hunk 2]
```diff
@@ -5,57 +5,35 @@
...
 
-#### Necessary Tools
+* `dotnet` command line tool
```
Comment (erickpintor) [inline README.md]: Should we add a link to it or just saying "`dotnet` command line tool" is sufficient for .NET developers to understand?

File: README.md [hunk 3]
```diff
@@ -5,57 +5,35 @@
...
 
-* Visual Studio 15
-* Nuget
-* Windows PowerShell (optional)
+### Build
 
-#### Installing Dependencies
+`dotnet build FaunaDB.Client`
```
Comment (erickpintor) [inline README.md]: I think it'd be better if stated as a command. Like "Run: `dotnet build FaunaDB.Client`".

File: README.md [hunk 4]
```diff
@@ -5,57 +5,35 @@
...
 
-`nuget restore faunadb-csharp.sln`
+if you're using MacOS or Linux you may need to override `FrameworkPathOverride` to point to the mono specific api like this:
```
Comment (erickpintor) [inline README.md]: "Like this" sounds too casual. I think you can just drop it.

File: README.md [hunk 5]
```diff
@@ -5,57 +5,35 @@
...
 
-#### Build
+`FrameworkPathOverride=/Library/Frameworks/Mono.framework/Versions/Current/lib/mono/4.5-api dotnet build FaunaDB.Client/ --framework net45`
 
-`& "C:\Program Files (x86)\MSBuild\14.0\Bin\amd64\MSBuild.exe" .\faunadb-csharp.sln /t:Build`
+### Running Tests
 
-#### Running Tests
+`dotnet test FaunaDB.Client.Test`
```
Comment (erickpintor) [inline README.md]: Same here: "Run: `dotnet test FaunaDB.Client.Test`."

File: README.md [hunk 6]
```diff
@@ -5,57 +5,35 @@
...
 ```xml
-<?xml version="1.0" encoding="utf-8"?>
-<packages>
-  <package id="FaunaDB.Client" version="1.0.0" targetFramework="net45" />
-</packages>
+<PackageReference Include="FaunaDB.Client" Version="2.0.0" />
 ```
 
-And then update your dependencies. See [Installing Dependencies](#installing-dependencies).
+or by using you IDE searching by `FaunaDB.Client`.
```
Comment (erickpintor) [inline README.md]: "or by using your IDE and searching for `FaunaDB.Client`"

File: README.md [hunk 7]
```diff
@@ -5,57 +5,48 @@
...
-### Mono
-
-#### Necessary Tools
+```bash
+dotnet build FaunaDB.Client
+```
 
-* Mono
-* Nuget
+if you're using MacOS or Linux you may need to override `FrameworkPathOverride` to point to the mono specific api:
```
Comment (eaceaser) [inline README.md]: capitalize M in mono

File: README.md [hunk 8]
```diff
@@ -5,57 +5,48 @@
...
 
-`mono packages/NUnit.ConsoleRunner.3.4.0/tools/nunit3-console.exe FaunaDB.Client.Test/bin/Debug/FaunaDB.Client.Test.dll --noresult`
+```bash
+FrameworkPathOverride=/Library/Frameworks/Mono.framework/Versions/Current/lib/mono/4.5-api dotnet test FaunaDB.Client.Test/ --framework net45
+```
 
 ## Referencing FaunaDB Assembly
 
-First install the Nuget package by editing your `packages.config`
+First install the Nuget package by adding the package reference on your MSBuild project:
```
Comment (eaceaser) [inline README.md]: on = to

File: README.md [hunk 9]
```diff
@@ -5,57 +5,48 @@
...
-#### Build
+Running the following command will run the tests for all supported .NET frameworks
 
-`xbuild faunadb-csharp.sln /t:Build`
+```bash
+dotnet test FaunaDB.Client.Test
+```
 
-#### Running Tests
+also, if you're using MacOS or Linux you may need to override `FrameworkPathOverride`:
```
Comment (eaceaser) [inline README.md]: You can drop the `also` and just say:

> If you're using macOS or Linux ...

File: README.md (hunk 7 above)
Comment (eaceaser) [reply README.md]: I think `MacOS` is officially stylized as `macOS` now

File: README.md [hunk 10]
```diff
@@ -5,57 +5,48 @@
 
 ## How to Build
 
-### Visual Studio
+### Requirements
 
-#### Necessary Tools
+* [.NET Sdk](https://www.microsoft.com/net/download/all)
+* Mono if you're using MacOS or Linux
```
Comment (eaceaser) [inline README.md]: i think `MacOS` is stylized as `macOS` now

File: README.md [hunk 11]
```diff
@@ -5,57 +5,48 @@
 
 ## How to Build
 
-### Visual Studio
+### Requirements
 
-#### Necessary Tools
+* [.NET Sdk](https://www.microsoft.com/net/download/all)
```
Comment (ashfire908) [inline README.md]: `SDK`?

File: README.md [hunk 12]
```diff
@@ -5,57 +5,48 @@
...
-### Mono
-
-#### Necessary Tools
+```bash
+dotnet build FaunaDB.Client
+```
 
-* Mono
-* Nuget
+if you're using MacOS or Linux you may need to override `FrameworkPathOverride` to point to the Mono specific api:
```
Comment (ashfire908) [inline README.md]: `If`

File: README.md [hunk 13]
```diff
@@ -5,57 +5,48 @@
...
-#### Build
+Running the following command will run the tests for all supported .NET frameworks
 
-`xbuild faunadb-csharp.sln /t:Build`
+```bash
+dotnet test FaunaDB.Client.Test
+```
 
-#### Running Tests
+if you're using macOS or Linux you may need to override `FrameworkPathOverride`:
```
Comment (ashfire908) [inline README.md]: `If`

**Pull request merged**